.github/ISSUE_TEMPLATE/account-recovery.yml

```
@@ -46,8 +47,15 @@ body:
     description: |
       By their nature account recovery issues are labor intensive. The PyPI team has limited resources available for their completion.
 
-      There may be a delay of a week or more between responses for these issues.
+      There may be a delay of few months or more between responses for these issues.
```

```suggestion
      There may be an extended delay between responses for these issues.
```

Let's not tie the message too closely to the current situation. "An extended delay" makes the point while not needing another change if things improve. But I will ask, does the suggestion read OK to a non-native speaker? I don't have a feel for whether "extended delay" conveys the same meaning as it does to a native speaker.

---

that works well for my french ears, thanks